### ISSUE TITLE
Hash Literals with a space

### ISSUE THREAD

**Issue by david-pm** [testdouble/standard#26]
First off, this library is a great idea!
Now for the complaints:
```
foo = { bar: "baz" }

Layout/SpaceInsideHashLiteralBraces: Space inside } detected.
```
I just can't live with: `{bar: "baz"}`. Just _look_ at it. I know Rails is not the gospel or anything but it is probably Ruby's most read codebase, and I only see hash literals with the space in there.

**Comment by searls** [comment]
Yes after a couple weeks of doing this I came to immediately appreciate this distinction. Did not expect to

**Comment by noelrappin** [comment]
I very much use the "space for blocks, no space for hashes rule" and would like to see it be part of standard.

**Comment by searls** [comment]
The yea's have it. Thanks for raising this, though, @david-pm!

**Comment by donapieppo** [comment]
In ruby 3 docs [class Hash](https://ruby-doc.org/3.2.2/Hash.html) we have
```ruby 
h = {:foo => 0, :bar => 1, :baz => 2}
```
while in ruby 2  docs [class Hash](https://ruby-doc.org/2.7.8/Hash.html) we had
```ruby
options = { font_size: 10, font_family: "Arial" }
```

**Comment by HLFH** [comment]
@donapieppo Related to this PR: https://github.com/ruby/ruby/pull/3056. Available since Ruby 3.0.

The cop policy for [`rubocop-rails-omakase`](https://github.com/rails/rubocop-rails-omakase/blob/main/rubocop.yml) is:

```
# Use `{ a: 1 }` not `{a:1}`.
# Use `{}` not `{  }`.
Layout/SpaceInsideHashLiteralBraces:
  Enabled: true
  EnforcedStyle: space
  EnforcedStyleForEmptyBraces: no_space
```

Since, there is no fully confirmed standard for Ruby styling, I'll use Omakase Ruby styling as default.

**Comment by searls** [comment]
Sorry, but after 6 years of working with Standard, I've come to really appreciate the visual cue that arrays/hashes are data literals because they lack space between their contents and the edge and blocks/lambdas are method-like because they do have a space. Unless you plan on putting a space at the beginning and end of an array literal (e.g. `[ 42 ]`), I can't see it as anything but inconsistent to insist spaces in hash literals.
